doc/api/vm.md

```diff
@@ -167,9 +167,49 @@ const contextifiedSandbox = vm.createContext({ secret: 42 });
     in stack traces produced by this Module.
   * `columnOffset` {integer} Spcifies the column number offset that is displayed
     in stack traces produced by this Module.
+  * `initalizeImportMeta` {Function} Called during evaluation of this Module to
+    initialize the `import.meta`. This function has the signature `(meta,
+    module)`, where `meta` is the `import.meta` object in the Module, and
+    `module` is this `vm.Module` object.
 
 Creates a new ES `Module` object.
 
+*Note*: Properties assigned to the `import.meta` object that are objects may
```

Comment:
i think people are already used to this just from using vm, might not need this example

Reply:
I think I would prefer to keep the example for now.